services/director-v2/src/simcore_service_director_v2/modules/dask_client.py

```diff
@@ -100,33 +104,37 @@ def _is_aborted_cb() -> bool:
         if remote_fct is None:
             remote_fct = comp_sidecar_fct
 
-        def _from_task_in_to_dask_resource(
-            task: TaskIn,
+        def _from_node_reqs_to_dask_resources(
+            node_reqs: NodeRequirements,
         ) -> Dict[str, Union[int, float]]:
-            reqs = task.runtime_requirements.upper().split(":")
-            return {r: 1 for r in reqs}
+            """Dask resources are set such as {"CPU": X.X, "GPU": Y.Y, "RAM": INT}"""
+            dask_resources = node_reqs.dict(exclude_unset=True, by_alias=True)
```

Comment:
These are labels, right? i mean, if we add in NodeRequirements new fields, these will still be compatible with "dask resources"? 
e.g. the new clusters will require new labels

Reply:
you are right. I will add a test.